lms/static/js/search/views/SearchListView.js

```diff
@@ -0,0 +1,97 @@
+;(function (define) {
```

Comment:
Thanks @dsego. I wonder if this isn't necessary now that https://github.com/edx/edx-platform/pull/6765 has landed (which adds RequireJS support directly into the LMS).